charts/fission-workflows/templates/deployment.yaml

```diff
@@ -35,6 +35,19 @@ spec:
           value: "{{ .Values.fnenv.fission.executor }}.{{ .Values.fnenv.fission.ns }}"
         - name: FNENV_FISSION_CONTROLLER
           value: "{{ .Values.fnenv.fission.controller }}.{{ .Values.fnenv.fission.ns }}"
+        readinessProbe:
+          httpGet:
+            path: "/status"
```

Comment:
Recommend changing this to /healthz for consistency with the rest of the Kubernetes world.